neqo-transport/src/send_stream.rs

```diff
@@ -596,6 +609,25 @@ impl SendStream {
         }
     }
 
+    pub fn bytes_written(&self) -> u64 {
```

Comment:
I think that you should document what these functions return a little more thoroughly.  They aren't 100% obvious.  (Don't rely on a link to the webtransport spec for the definitions, though the text is pretty good and so it can probably be copied over.)